webapp/channels/src/components/common/svg_images_components/contact_us_svg.tsx

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+type SvgProps = {
+    width: number;
+    height: number;
+}
+
+const ContactUsSvg = (props: SvgProps) => (
+    <svg
+        width={props.width ? props.width.toString() : '125'}
+        height={props.height ? props.height.toString() : '97'}
+        viewBox='0 0 125 97'
+        fill='none'
+        xmlns='http://www.w3.org/2000/svg'
+    >
+        <g clipPath='url(#clip0_4842_184301)'>
+            <rect
+                opacity='0.08'
+                width='50'
+                height='50'
+                rx='2'
+                fill='#1C58D9'
```

Comment:
@nickmisasi @matthewbirtch for this pricing modal svg, do we need to use the theme variables?

Reply:
Actually, looking at the comment in Figma, it seems that it's the same we use for the Threads empty state here that is thremed: https://github.com/mattermost/mattermost/blob/master/webapp/channels/src/components/threading/common/chat_illustration.tsx#L6

@nickmisasi not sure if you want to move that to a shared Svg location to share it in both places ?